lib/lib-storage/package.json

```
@@ -23,6 +23,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "buffer": "^5.6.0",
+    "events": "^3.3.0",
     "stream-browserify": "^3.0.0",
     "tslib": "^2.3.0"
```

Use exact versions of non-tslib dependencies and update `yarn.lock`?
```suggestion
    "buffer": "5.6.0",
    "events": "3.3.0",
    "stream-browserify": "3.0.0",
    "tslib": "^2.3.0"
```